diff --git a/Makefile b/Makefile
index 9428506..8518fdd 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,8 @@
 #Under BSD License
 #See clock.c for the license detail.
 
-SRC = ttyclock.c ttyclock.h
+SOURCE = ttyclock.c
+HEADER = ttyclock.h
 CC ?= gcc
 BIN ?= tty-clock
 PREFIX ?= /usr/local
@@ -26,10 +27,10 @@ else
 	LDFLAGS += $$(pkg-config --libs ncurses)
 endif
 
-tty-clock : ${SRC}
+tty-clock : ${SOURCE} ${HEADER}
 
-	@echo "building ${SRC}"
-	${CC} ${CFLAGS} ${SRC} -o ${BIN} ${LDFLAGS}
+	@echo "building ${SOURCE}"
+	${CC} ${CFLAGS} ${SOURCE} -o ${BIN} ${LDFLAGS}
 
 install : ${BIN}
 

